components/toast/index.css

```diff
@@ -156,45 +150,34 @@ governing permissions and limitations under the License.
   flex: 0 0 auto;
   align-items: flex-start;
 
-  margin-block-start: var(--mod-toast-spacing-top-edge-to-divider, var(--spectrum-toast-spacing-top-edge-to-divider));
-  margin-block-end: var(--mod-toast-spacing-bottom-edge-to-divider, var(--spectrum-toast-spacing-bottom-edge-to-divider));
-
-  /* Divider color for Text | Action Button */
-  border-inline-start-color: var(--mod-toast-divider-color, var(--spectrum-toast-divider-color));
-  padding-inline-end: var(--mod-toast-spacing-close-button, var(--spectrum-toast-spacing-close-button));
+  margin-block-start: var(--mod-toast-spacing-top-edge-to-close-button, var(--spectrum-toast-spacing-top-edge-to-close-button));
+  margin-block-end: var(--mod-toast-spacing-bottom-edge-to-close-button, var(--spectrum-toast-spacing-bottom-edge-to-close-button));
+  padding-inline-start: var(--mod-toast-spacing-text-to-close-button, var(--spectrum-toast-spacing-text-to-close-button));
+  padding-inline-end: var(--mod-toast-spacing-close-button-to-edge, var(--spectrum-toast-spacing-close-button-to-edge));
 
   .spectrum-CloseButton {
     align-self: flex-start;
   }
 }
 
+.spectrum-Toast:has(.spectrum-Button) {
+  .spectrum-Button {
+    margin-inline-start: var(--mod-toast-spacing-text-to-action-button-horizontal, var(--spectrum-toast-spacing-text-to-action-button-horizontal));
+  }
+
+  .spectrum-Toast-buttons {
+    padding-inline-start: var(--mod-toast-spacing-action-button-to-close-button, var(--spectrum-toast-spacing-action-button-to-close-button));
```

Comment:
This is a great callout! TLDR; design has approved `spacing-200` in this case.

Some context: Patrick and I chatted with design about this and agreed with them that the spacing will always be `spacing-200` when an action button is used in Toast. This is because we can't tell using CSS and flexbox when the action button wraps to the next line, so there's not a good way (in CSS) to have `spacing-300` in this case and `spacing-200` if the button isn't wrapped.